ibis/expr/types/relations.py

```diff
@@ -170,14 +171,15 @@ def _resolve(self, exprs):
 
     # TODO(kszucs): should be removed
     def _ensure_expr(self, expr):
+        if isinstance(expr, Deferred):
+            return expr.resolve(self)
         if isinstance(expr, str):
             return self[expr]
-        elif isinstance(expr, (int, np.integer)):
```

Comment:
I tend to prefer more explicit branches using `elif` and `else`.

Reply:
Done.